ci/kind/test-e2e-kind.sh

```diff
@@ -239,6 +239,13 @@ fi
 if $flow_visibility; then
     manifest_args="$manifest_args --feature-gates FlowExporter=true,L7FlowExporter=true --extra-helm-values-file $FLOW_VISIBILITY_HELM_VALUES"
 fi
+if $deploy_no_antrea_worker; then
+    common_prefix="affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0]"
+    manifest_args="$manifest_args --extra-helm-values agent.$common_prefix.key=no-antrea"
+    manifest_args="$manifest_args --extra-helm-values agent.$common_prefix.operator=DoesNotExist"
+    manifest_args="$manifest_args --extra-helm-values controller.$common_prefix.key=no-antrea"
+    manifest_args="$manifest_args --extra-helm-values controller.$common_prefix.operator=DoesNotExist"
+fi
```

Comment:
it would probably be better to use a file, and check it in under `ci/kind`

Reply:
Will do